README.md

```diff
@@ -1 +1,145 @@
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 Budget App <a name="about-project"></a>
+
+- This is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- [Ruby](https://rubocop.org/)
+- Ruby on rails
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- storing all the data in the database using PostgreSQL DBMS
+- A user can create a category.
+- A user can add transcations to a given category, displaying the total amount spent.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Use the following instructions
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Install [ruby](https://rubocop.org/)
+
+### Setup
+
+- Clone the project
+```
+git clone https://github.com/Omwomo/budget-app.git .
+```
+to the directory of your desire
+
+- To create database run
+```
+rails db:create
+rails db:migrate
+```
+
+- Set up your [linter](https://github.com/microverseinc/linters-config/blob/master/ruby/) testing file
+
+### Install
+
+To retrieve the dependencies of this project run
+```
+ bundle install
+```
+
+### Usage
+
+To make use of this project on your local computer
+```
+rails s
+```
+
+## 👥 Author <a name="author"></a>
+
+👤 **Eugene Omwomo**
+
+- GitHub: [@Omwomo](https://github.com/Omwomo)
+- Twitter: [@eugeneoduor9](https://twitter.com/eugeneoduor9)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/Omwomo)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features
+- Deploy the app.
+- Add tracking to the expenditure.
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Omwomo/budget-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please don't hesitate to support me with your valued contribution and don't forget to star the repo to reach more
+people.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank:
+- Microverse Review Team
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ <a name="faq"></a>
+
+Can I clone this project ?\*\*
+
+- Of course you can as long as you fork it and star it
+
+Can I contribute to this project?\*\*
+
+- definitely you can
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](MIT.md) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
##### **Design**

- [ ] Can you follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
*Typography: font face, size and weight.
*Layout: composition and space between elements.Design

👉  This requirement concerns only the `Sign up` and `Login` pages

- Your Designs
- Log in
![Capture](https://github.com/Omwomo/budget-app/assets/63972670/d9d848d4-5086-4459-920f-9b2a8aa1cc86)
-
- Sign up
![Capture1](https://github.com/Omwomo/budget-app/assets/63972670/9bc6ea2a-5a37-4908-a3d4-d6395dab6b64)


- [ ] **Original designs:** 
-
![Capture](https://github.com/Omwomo/budget-app/assets/63972670/b0183c0e-bdca-4cac-9db6-96d9c82650d6)
-
![Capture1](https://github.com/Omwomo/budget-app/assets/63972670/d6def2bd-0f7b-47ba-b4f4-d06a446a9ebb)

.

Reply:
well resolved! ✔️